app/src/state/rangeInput.tsx

```diff
@@ -32,6 +36,8 @@ export function RangeInputMolecule(
     name,
     minAtom,
     maxAtom,
+    sliderMinAtom,
+    sliderMaxAtom,
```

Comment:
As suggested above, we can pass in plain numbers here (not atoms).
```suggestion
    min,
    max,
```